src/components/users/userProfile/UserProfile.js

```diff
@@ -37,6 +37,7 @@ const UserProfile = () => {
     if (id === loggedInId) {
       return setCheckId(true);
     }
+    setCheckId(false)
```

Comment:
Interesting that this state doesn't return false by default actually. Also do we need to add a return at the start of line 40?

Reply:
I think by default when the 'if' statement fires, any block of code after it will not run, IE line 40, providing there is a return statement inside, which there is. The code functions with or without a return on line 40, so this is a great shout, unsure which is better practice.